utils/pckgen

```diff
@@ -0,0 +1,454 @@
+#!/bin/bash
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+# To make additional packages creation possible
+# through this script all you need to do is:
+# - add 'case' command to set variable 'GENERATE_<NAME>=1'
+# - add function 'generate_<name>' to handle the task
+# - add 'DEPENDENCIES_<NAME>' array if needed
+# - update help!
+
+
+NAME="open-cas-linux"
+# TODO: change 'supported from' to 20.3
+SUPPORTED_FROM_VERSION="20.1"
```

Comment:
I suggest to :
1. Change current version in CAS_VERSION_GEN to 20.3
2. Change this variable to 20.3